### PR TITLE
feat: table component for sessions

### LINKED ISSUE #808: 🚀 [Feature Request] - Proper table to handle and show data in sessions
### Is your feature request related to a problem ? Please describe. ✍️

The sessions table is not optimized to handle and display data .

We can use packages like tanstack react table to achieve the desired functionality.

### Describe briefly how you would implement this enhancement.

```bash
Some steps to implement this enhancement-
1. Step 1
2. Step 2
3. Step 3
```

### Additional context or screenshots

image

### Are you contributing for a program?

Other

### Code of Conduct

- [X] I have read and agree to the Contributing Guidelines of this project.

### PR DESCRIPTION
**Pull request by rishuraj1**
## Related Issue

Closes: #808 

## Description of Changes

- created a table component to handle session data for mentor dashboard
- made it responsive

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Screenshots

image

### CODE REVIEW

**Comment by anmode** [comment]
@rishuraj1 is that possible we can stick to Previous UI?? using table component

**Comment by rishuraj1** [comment]
@anmode  Yes, okay I will do it now.

**Comment by rishuraj1** [comment]
@anmode 

image

image

**Comment by anmode** [comment]
Align is not good!  and UI is not same. let me show 
Screenshot 2023-11-08 at 7 19 07 PM

see the heading is not bold white... and in your UI that is bold white and other is greyish entry!
and rest is fine

File: components/mentorDashboard/Bookings.js [hunk 1]
```diff
@@ -81,7 +82,13 @@ const Bookings = ({ setLoadingState, setErrorState }) => {
...
       {/* sessions list */}
       <div className="tw-mt-8">
-        <ul className="tw-bg-primary-100 tw-text-white  tw-flex tw-items-center tw-gap-6 tw-p-6 tw-border tw-border-b-0 tw-rounded-t">
+        {sessions ? (
+          <TableComponent sessions={sessions} activeTab={activeTab} />
+        ) : (
+          <Spinner />
+        )}
+
+        {/* <ul className="tw-bg-primary-100 tw-text-white  tw-flex tw-items-center tw-gap-6 tw-p-6 tw-border tw-border-b-0 tw-rounded-t">
```
Comment (anmode) [inline components/mentorDashboard/Bookings.js:91]: can you same implement for user dashboard to use this table component in case of bookings please!

**Review by anmode — CHANGES_REQUESTED** [review]
last thing

File: components/mentorDashboard/Bookings.js (hunk 1 above)
Comment (rishuraj1) [reply components/mentorDashboard/Bookings.js:91]: yes sure will be done by tomm.

**Comment by rishuraj1** [comment]
@anmode  done, 
image

**Comment by anmode** [comment]
last thing..these heading are in white bold.. please make it normal! as i show in previous UI
and i see alignment is also not proper:
Screenshot 2023-11-09 at 5 42 47 AM

**Comment by anmode** [comment]
One more thing Queries page needs to align properly to use whole space: Need a little designing!
Screenshot 2023-11-09 at 5 44 06 AM


Rest whole dashboard is awesome now!

**Comment by rishuraj1** [comment]
@anmode  
image

image

**Comment by anmode** [comment]
looks cool but color kaise change hogyaa.. I mean theme color was good :)

**Comment by rishuraj1** [comment]
@anmode  oops, my bad didn't notice 😅, fixed it now.

**Comment by anmode** [comment]
> @anmode oops, my bad didn't notice 😅, fixed it now.

no worries i think now it is good to merge this PR! Yeah!!!

**Pull request merged**